### ISSUE TITLE
README information shown in VS Code should be less build focused

### ISSUE THREAD

**Issue by stevedlawrence** [apache/daffodil-vscode#100]
The README that is shown in the VS Code app is the one that is at the root of our repo, which is focused on how to build the vsix file from source. But since the extension is already installed, then this README information isn't very helpful. We already have different LICENSE and NOTICE files that go in our vsix file. Should we also have a different README file that is more focused on how to use the debugger instead of how to build it, making it more useful to people viewing the README in VS Code?

**Comment by hdalsania** [comment]
@stevedlawrence  You are correct, PR#174 was not merged but Readme file is fixed to provide links to development and usage information wiki and instructions to download vsix file from marketplace with issue #411 and PR #424
